cmd/emailreport/Gopkg.lock

```diff
@@ -0,0 +1,15 @@
+# This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
+
+
+[[projects]]
+  name = "github.com/BurntSushi/toml"
+  packages = ["."]
+  revision = "b26d9c308763d68093482582cea63d69be07a0f0"
```

Comment:
until fairly recently `toml` used a license (the WTF license) that we'd rather not have in our project - but, then they updated it :-) Can we thus move the revision/version of `toml` we are tracking, maybe to be the same as that in `checkmetrics` for instance:
https://github.com/clearcontainers/tests/blob/master/cmd/checkmetrics/Gopkg.toml#L24

In fact, generally I believe for our projects, we set `dep` to track the `master` branches rather than fixed versions (unless we have to pin/regress to fix an issue for instance).